Extensions/xkit_patches.js

```diff
@@ -219,7 +219,7 @@ XKit.extensions.xkit_patches = new Object({
 							XKit.interface.kitty.set(e.data.response.headers["x-tumblr-kittens"]);
 						}
 
-						if (e.data.success) {
+						if (e.data.success && 200 <= e.data.response.status < 300) {
```

Comment:
`200 <= var < 300` does not do what you think it does:

```
> 10 <= 4000 < 5
true
> 10 <= 4000
true
> true < 5
true
```

instead, let's do something like:

```js
let {response, success} = e.data;
if (success && response.status >= 200 && response.status < 300) {
```